pom.xml

```diff
@@ -219,8 +214,24 @@
                     <groupId>commons-collections</groupId>
                     <artifactId>commons-collections</artifactId>
                 </exclusion>
+                <exclusion>
+                    <groupId>org.owasp.antisamy</groupId>
+                    <artifactId>antisamy</artifactId>
+                </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+          <groupId>org.apache.xmlgraphics</groupId>
+          <artifactId>batik-css</artifactId>
+          <version>1.14</version>
+          <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.w3c.css</groupId>
```

Comment:
It should be embedded if this is necessary

Reply:
I think that should already be the cause, but probably is not clear enough. All that is needed for the Sling Starter to deploy the new version of the Sling XSS bundle is at https://github.com/apache/sling-org-apache-sling-starter/compare/master...rombert:sling-org-apache-sling-starter:master .